src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java

```diff
@@ -204,6 +208,17 @@ else if (canFeedChildren() && numOfFeedableAnimals > 0)
         return START_WORKING;
     }
 
+    /**
+     * Checks if we can breed this entity
+     *
+     * @param entity
+     * @return
+     */
+    protected boolean isBreedAble(final AnimalEntity entity)
```

Comment:
the uppercase "A" is weird =D

Reply:
and @return true if breedable.

Reply:
intellij complains with small A though so I guess its correct xD